Makefile

```diff
@@ -1,11 +1,12 @@
-.PHONY: lint format dev-setup dev-install prod-install
+.PHONY: lint format dev-setup dev-install prod-install test install-test-requirements
 
 # Commands
 ## Python
 flake8	= flake8 python/fastexcel *.py
 isort	= isort python/fastexcel *.py
 black	= black python/fastexcel *.py
 mypy	= mypy python/fastexcel *.py
+pytest	= python -m pytest -vvv -s python/tests
```

Comment:
I tend to prefer adding the tests path in [pyproject.toml](https://docs.pytest.org/en/6.2.x/customize.html#pyproject-toml). But it's nice to not enforce the extra options as sometimes people want to use `-s`, `-x` or nothing at all